ts/Core/Series/Series.ts

```diff
@@ -4864,6 +4866,7 @@ class Series {
         }
 
         const sharedClipKey = [
+            this.useTemporaryClip ? '_temporaryClip' : '', // #4406
```

Comment:
You could probably get away with only adding a temp prefix if a shared clip with that key already exists.